src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -8,5 +8,6 @@
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_TASK_ID = "The task ID provided is invalid";
```

Comment:
use Task.MESSAGE_INVALID_ID instead

Reply:
Sure. These messages are scattered all around the code base. We should figure a way to consolidate them in the future.